submodules/bastion/main.tf

```diff
@@ -22,6 +22,17 @@ resource "aws_security_group" "bastion" {
   }
 }
 
+resource "aws_security_group_rule" "bastion_outbound" {
```

Comment:
Should we also make the egress configurable?

Reply:
I'm not sure, do you think it's ever going to be changed? In the past I've never heard of us needing to change it for security or other reasons